include/gpgmm_d3d12.h

```diff
@@ -181,7 +181,8 @@ namespace gpgmm::d3d12 {
     };
     \endcode
     */
-    using CreateHeapFn = std::function<HRESULT(ID3D12Pageable** ppPageableOut)>;
+
```

Comment:
Leaving a TODO to update README.md as well.

Reply:
Updated with the new CreateHeap signature. It might be a little confusing without looking at the explanation in the header. Happy to add more detail if you like.